bot.go

```diff
@@ -34,19 +34,36 @@ type BotAPI struct {
 //
 // It requires a token, provided by @BotFather on Telegram.
 func NewBotAPI(token string) (*BotAPI, error) {
-	return NewBotAPIWithClient(token, &http.Client{})
+	return CreateNewBotApi(token, &http.Client{}, false)
 }
 
 // NewBotAPIWithClient creates a new BotAPI instance
 // and allows you to pass a http.Client.
 //
 // It requires a token, provided by @BotFather on Telegram.
 func NewBotAPIWithClient(token string, client *http.Client) (*BotAPI, error) {
+	return CreateNewBotApi(token, client, false)
+}
+
+// NewBotAPIWithClient creates a new BotAPI instance
```

Comment:
Comment needs to be fixed.

```suggestion
// NewBotAPIWithDebug creates a new BotAPI instance
```